src/binary_annotator.cpp

```diff
@@ -34,6 +34,14 @@ static BinarySection MakeBinarySection(
   return section;
 }
 
+static BinarySection MakeSingleRegionBinarySection(const std::string &name,
+                                                   const BinarySectionType type,
+                                                   const BinaryRegion &region) {
+  std::vector<BinaryRegion> regions;
+  regions.emplace_back(region);
```

Comment:
nit: prefer push_back over emplace_back if they both work for the same arguments

(I mentioned this in the last review but forgot to didn't un-resolve the comment so I'm not sure if you saw and intentionally ignored it)

https://abseil.io/tips/112

Reply:
Done.